source/developer-guides/boot-flow.rst

```diff
@@ -36,6 +36,117 @@ Payload    Load, verify and launch OS images; or perform firmware update
   If verification fails, one of two things occurs. If resiliency is enabled, the boot is swapped over to the alternate partition
   to recover the broken partition (see :ref:`firmware-resiliency-and-recovery`). Otherwise, the boot is halted.
 
+Stage 1A
+~~~~~~~~
+

```

Comment:
Please add this back - When coming out of reset, the processor may be in a very raw state and needs some basic
initialization before the rest of the platform can be initialized.